src/packages/database/database_manager.py

```diff
@@ -135,3 +183,6 @@ def __init__(self,
         self._dbpool.putconn(connection)
 
         return list(result)
+
+
+plugin_manager.register(DatabaseManager, "Database")
```

Comment:
[Question] For consistency's sake, most modules seem to be registering through their `__init__.py` files. Should we stick to that standard?

Reply:
yes. will move.